python-types.c

```diff
@@ -1460,7 +1469,7 @@ tgl_Msg_repr(tgl_Msg *self)
 #else
   ret = PyUnicode_FromFormat("<tgl.Msg id=%ld, flags=%d, mention=%R, out=%R, unread=%R, service=%R, src=%R, "
                              "dest=%R, text=%R, media=%R, date=%R, fwd_src=%R, fwd_date=%R, reply_id=%R, reply=%R>",
-                             self->msg->id, self->msg->flags,
+                             self->msg->permanent_id.id, self->msg->flags,
```

Comment:
Tested with ubuntu's 14.04 default python 2.7 and my support for bot keyboard (reply_markup) — all works well.

P.S. Question to universe: will somebody merge any PR in this project? :(